# Author: Damyan Ivanov <dmn@debian.org>
# Description: include the right header file
#  GCC 4.3 has stricter rules about inclusion propagation
--- a/src/translator_manager.cpp
+++ b/src/translator_manager.cpp
@@ -22,7 +22,7 @@
 
 #include <sys/types.h>
 #include <dirent.h>
-#include <string>
+#include <cstring>
 #include <iostream>
 #include <algorithm>
 
